GitUI/Avatars/IAvatarDownloader.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Drawing;
+using System.Threading.Tasks;
+
+namespace GitUI.Avatars
+{
+    public interface IAvatarDownloader
+    {
+        Task<Image> DownloadImageAsync(Uri imageUrl);
+    }
```

Comment:
I think we could use a more abstract interface like IImageDownloader or IDownloader.

Reply:
What would be get with that?

Reply:
Image, byte[], Stream...

Reply:
Most likely we have or will have more than one type of resource that requires downloading.

Reply:
I could try to generalize it even more but I don't think making it "more abstract" would be a good fit for this PR. Feels a bit like a rabbit hole to me 😕